api/config.go

```diff
@@ -18,17 +18,32 @@ type ConfigUnset struct {
 
 // Config is the structure of an app's config.
 type Config struct {
-	Owner       string                  `json:"owner,omitempty"`
-	App         string                  `json:"app,omitempty"`
-	Values      map[string]interface{}  `json:"values,omitempty"`
-	Memory      map[string]interface{}  `json:"memory,omitempty"`
-	CPU         map[string]interface{}  `json:"cpu,omitempty"`
+	// Owner is the app owner. It cannot be updated with config.Set(). See app.Transfer().
+	Owner string `json:"owner,omitempty"`
+	// App is the app name. It cannot be update at all at this moment.
+	App string `json:"app,omitempty"`
+	// Values are exposed as environmantal variables to the app.
+	Values map[string]interface{} `json:"values,omitempty"`
+	// Memory is used to set process memory limits. The key is the process name
+	// and the value is a number followed by a memory unit (G, M, K, or B). Ex: 200G
+	Memory map[string]interface{} `json:"memory,omitempty"`
```

Comment:
I'm interested in doing so, but it's out of the score of this PR. We should open an issue on it.